src/routes/healthcheck.js

```diff
@@ -0,0 +1,10 @@
+const cors = require('cors');
+const path = require('../services/path');
+
+module.exports = function HealthCheck(app, opts) {
+  this.perform = () => {
+    app.get(path.generate('healthcheck', opts), cors(), (request, response) => {
```

Comment:
If it is a simple request you won't need the `cors` middleware:
```suggestion
    app.get(path.generate('healthcheck', opts), (request, response) => {
```

Reply:
Nope because the `origin:*` header is needed on the client-side.
Without it, the browser does not let JS access the response (neither the status code)